observability/index.adoc

```diff
@@ -38,14 +38,14 @@ For more information, see xref:../logging/cluster-logging.adoc#cluster-logging[A
 == Distributed tracing
 Store and visualize large volumes of requests passing through distributed systems, across the whole stack of microservices, and under heavy loads. Use it for monitoring distributed transactions, gathering insights into your instrumented services, network profiling, performance and latency optimization, root cause analysis, and troubleshooting the interaction between components in modern cloud-native microservices-based applications.
 
-For more information, see xref:../distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc#distributed-tracing-architecture[Distributed tracing architecture].
+For more information, see xref:distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc#distributed-tracing-architecture[Distributed tracing architecture].
```

Comment:
I think it would be more correct per our guidelines to include the full new path for both of these:
```suggestion
For more information, see xref:../observability/distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc#distributed-tracing-architecture[Distributed tracing architecture].
```

Reply:
@adellape, thank you for your feedback. :slightly_smiling_face: I think I'm okay here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#internal-cross-references.